python/lsst/pipe/tasks/multiBand.py

```diff
@@ -763,33 +780,68 @@ def mergeCatalogs(self, catalogs, patchRef):
         # This first zip iterates over all the catalogs simultaneously, yielding a sequence of one
         # record for each band, in order.
         for n, orderedRecords in enumerate(zip(*orderedCatalogs)):
```

Comment:
It's not strictly part of this change, but since ``n`` is unused, can we get rid of the enumeration here?